### PR TITLE
screenkey: add page

### PR DESCRIPTION
**Pull request by ashishbinu**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/screenkey.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Display your keys and mouse buttons pressed on the screen:
+
+`screenkey --mouse`
+
+- Launch settings menu of screenkey:
+
+`screenkey --show-settings`
+
+- Change the postion(top/center/bottom/fixed) of screenkey:
```
Comment (bl-ue) [inline pages/linux/screenkey.md:18]:
```suggestion
- Change the position (top/center/bottom/fixed) of screenkey:
```

**Comment by CleanMachine1** [comment]
Hi @ashishbinu thanks for your first PR, this process till merging may take hours or it may take take days depending on how many errors there maybe, please stay with each suggestion and give your opinions/commit them

File: pages/linux/screenkey.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`screenkey --position top`
+
+- Change the modifiers displayed on the screen like ctrl to ⌘ in mac mode:
+
+`screenkey --mods-mode mac`
+
+- Change the appearance of screenkey:
+
+`screenkey --bg-color '#353535' --font Hack --font-color yellow --opacity 0.8`
```
Comment (bl-ue) [inline pages/linux/screenkey.md:28]:
```suggestion
`screenkey --bg-color "{{#353535}}" --font {{Hack}} --font-color {{yellow}} --opacity {{0.8}}`
```

File: pages/linux/screenkey.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Drag and select a window on screen to display screenkey:
+
+`screenkey --position fixed --geometry $(slop -n -f '%g')`
```
Comment (bl-ue) [inline pages/linux/screenkey.md:32]:
```suggestion
`screenkey --position fixed --geometry {{$(slop -n -f '%g')}}`
```

File: pages/linux/screenkey.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+`screenkey --mouse`
+
+- Launch settings menu of screenkey:
+
+`screenkey --show-settings`
+
+- Change the postion(top/center/bottom/fixed) of screenkey:
+
+`screenkey --position top`
+
```
Comment (CleanMachine1) [inline pages/linux/screenkey.md:21]:
```suggestion
- Change the postion of screenkey:

`screenkey --position {{top|center|bottom|fixed}}`

```

File: pages/linux/screenkey.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+- Change the modifiers displayed on the screen like ctrl to ⌘ in mac mode:
+
+`screenkey --mods-mode mac`
```
Comment (bl-ue) [inline pages/linux/screenkey.md:24]:
```suggestion
- Change the format of the key modifiers displayed on screen:

`screenkey --mods-mode {{normal|mac|emacs}}`
```

File: pages/linux/screenkey.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Launch settings menu of screenkey:
+
+`screenkey --show-settings`
+
+- Change the postion(top/center/bottom/fixed) of screenkey:
+
+`screenkey --position top`
+
+- Change the modifiers displayed on the screen like ctrl to ⌘ in mac mode:
```
Comment (CleanMachine1) [inline pages/linux/screenkey.md:22]:
```suggestion
- Change the modifiers displayed on the screen like Ctrl to ⌘ in mac mode:
```

File: pages/linux/screenkey.md (hunk 2 above)
Comment (CleanMachine1) [inline pages/linux/screenkey.md:28]:
```suggestion
`screenkey --bg-color '#{{353535}}' --font {{font_name}} --font-color {{color}} --opacity {{0.8}}`
```

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Comment by bl-ue** [comment]
@CleanMachine1 I submitted those suggestions an hour ago — do you mind if I hide yours that conflict with mine?

File: pages/linux/screenkey.md (hunk 2 above)
Comment (CleanMachine1) [reply pages/linux/screenkey.md:28]: Most of these are in my changes

Comment (bl-ue) [reply pages/linux/screenkey.md:28]: It's better to change the single quotes to double and include the hashtag.

**Comment by CleanMachine1** [comment]
If yours are better sure

File: pages/linux/screenkey.md (hunk 5 above)
Comment (ashishbinu) [reply pages/linux/screenkey.md:24]: I added all the available options below. Should I commit all the suggested  changes as a single commit if everything looks good?
```suggestion
- Change the format of the key modifiers displayed on screen:

`screenkey --mods-mode {{normal|emacs|mac|win|tux}}`
```

**Comment by CleanMachine1** [comment]
Unaware why the workflow didn't run after commits, it was approved

File: pages/linux/screenkey.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# screenkey
+
+> A screencast tool to display your keys.
+> More information: <https://www.thregr.org/~wavexx/software/screenkey/>.
+
+- Display your keys pressed on the screen:
```
Comment (CleanMachine1) [inline pages/linux/screenkey.md:6]:
```suggestion
- Display keys which are currently being pressed on the screen:
```

File: pages/linux/screenkey.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+`screenkey`
+
+- Display your keys and mouse buttons pressed on the screen:
```
Comment (CleanMachine1) [inline pages/linux/screenkey.md:10]:
```suggestion
- Display keys and mouse buttons which are currently being pressed on the screen:
```

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]
We tend not to use you and your

File: pages/linux/screenkey.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+`screenkey --mouse`
+
+- Launch settings menu of screenkey:
```
Comment (CleanMachine1) [inline pages/linux/screenkey.md:14]:
```suggestion
- Launch the settings menu of screenkey:
```

File: pages/linux/screenkey.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
+# screenkey
+
+> A screencast tool to display your keys.
```
Comment (bl-ue) [inline pages/linux/screenkey.md:3]: What do you think about this, @ashishbinu?
```suggestion
> A screencast tool to display keys currently being pressed.
```

Comment (ashishbinu) [reply pages/linux/screenkey.md:3]: How about this?

```suggestion
> A screencast tool to display the keys pressed.
```

Comment (bl-ue) [reply pages/linux/screenkey.md:3]: Sounds better than mine, except would sound even bit better if you removed `the`, like this:
```suggestion
> A screencast tool to display keys pressed.
```

Comment (marchersimon) [reply pages/linux/screenkey.md:3]: I don't know how much value we want to give this, but I think `the keys pressed` may be a bit easier to read for non-native speakers.

Comment (bl-ue) [reply pages/linux/screenkey.md:3]: Well, it just doesn't sound very good, and I can't exactly put my finger on it, but the subject "keys pressed" seems a bit too small for `the` 🤔

File: pages/linux/screenkey.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+- Display keys and mouse buttons which are currently being pressed on the screen:
+
+`screenkey --mouse`
+
+- Launch the settings menu of screenkey:
+
+`screenkey --show-settings`
+
+- Change the position of screenkey:
```
Comment (marchersimon) [inline pages/linux/screenkey.md:18]: Does it open a new window at this position or does it change the position of a running screenkey instance?

Comment (ashishbinu) [reply pages/linux/screenkey.md:18]: @marchersimon It creates a new instance and places it on the given position. If there is a running instance in the background, running `screenkey --position top` will create a new screenkey instance at specified position i.e. there would be two instances running simultaneously.

Comment (marchersimon) [reply pages/linux/screenkey.md:18]: Then how about:
```suggestion
- Launch screenkey at a specific position:
```

Comment (ashishbinu) [reply pages/linux/screenkey.md:18]: How about the above change @CleanMachine1 @bl-ue ?

Comment (ashishbinu) [reply pages/linux/screenkey.md:18]: How about the above change @CleanMachine1 @bl-ue ?

Comment (CleanMachine1) [reply pages/linux/screenkey.md:18]: Please dont ping us twice.

Comment (bl-ue) [reply pages/linux/screenkey.md:18]: It was an accident, I believe @CleanMachine1 :)

Comment (bl-ue) [reply pages/linux/screenkey.md:18]: Looks good to me @ashishbinu!

Comment (ashishbinu) [reply pages/linux/screenkey.md:18]: > Please dont ping us twice.

Sorry I didn't know it was sent twice.My internet was kinda slow when I tried to send it.

File: pages/linux/screenkey.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+`screenkey --position {{top|center|bottom|fixed}}`
+
+- Change the format of the key modifiers displayed on screen:
+
+`screenkey --mods-mode {{normal|emacs|mac|win|tux}}`
+
+- Change the appearance of screenkey:
+
+`screenkey --bg-color "{{#353535}}" --font {{Hack}} --font-color {{yellow}} --opacity {{0.8}}`
```
Comment (marchersimon) [inline pages/linux/screenkey.md:28]:
```suggestion
`screenkey --bg-color "{{#3b3b3b}}" --font {{Hack}} --font-color {{yellow}} --opacity {{0.8}}`
```
A hex code without a single letter kind of doesn't look right 😄

File: pages/linux/screenkey.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+- Drag and select a window on screen to display screenkey:
+
+`screenkey --position fixed --geometry {{$(slop -n -f '%g')}}`
```
Comment (marchersimon) [inline pages/linux/screenkey.md:32]: > screenkey: error: argument -g/--geometry: invalid geometry value: '{{330x157+1314+533}}'

Doesn't really work for me. I can select an area, but then I get the above error.

Comment (ashishbinu) [reply pages/linux/screenkey.md:32]: The double curly brackets `{{}}` should not be included when the command is run in the terminal. It is a token syntax to show user provided values which tldr clients will need to highlight it.

Comment (marchersimon) [reply pages/linux/screenkey.md:32]: Being a tldr maintainer for a few months and still forgetting that `{{ }}` is tldr-only syntax... 😄

File: pages/linux/screenkey.md (hunk 12 above)
Comment (CleanMachine1) [reply pages/linux/screenkey.md:28]: I think it should be #FFFFFF

Comment (marchersimon) [reply pages/linux/screenkey.md:28]: How would that be a benefit? I think `3b3b3b` is astethically more pleasing than `FFFFFF`.

Comment (bl-ue) [reply pages/linux/screenkey.md:28]: How about `#A1B2C3`?

Comment (marchersimon) [reply pages/linux/screenkey.md:28]: This would prabably be the most generic variant. However I prefer `#a1b2c3`.

Comment (ashishbinu) [reply pages/linux/screenkey.md:28]: Yeah, I also prefer `#a1b2c3`.
```suggestion
`screenkey --bg-color "{{#a1b2c3}}" --font {{Hack}} --font-color {{yellow}} --opacity {{0.8}}`
```

**Review by marchersimon — APPROVED** [review]
Looks good, thanks @ashishbinu

File: pages/linux/screenkey.md (hunk 11 above)
Comment (CleanMachine1) [reply pages/linux/screenkey.md]: Ah, ok, just sent 2 requests. No harm! Just seemed like you were spamming a little :+1:

File: pages/linux/screenkey.md (hunk 12 above)
Comment (CleanMachine1) [reply pages/linux/screenkey.md]: That auto hex to color is pretty nice!

**Review by CleanMachine1 — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**